### PR TITLE
Show flag specific name in results.html

### PR DESCRIPTION
**Pull request by WeizhongX**
This now works in RWT but not in wptrunner. The reason is flag_name is not properly set in the final json file.

Bug: chromium: 1426296

### CODE REVIEW

**Review by jonathan-j-lee — APPROVED** [review]

**Comment by WeizhongX** [comment]
@jcscottiii do you think if we can admin merge this?

This is a simple change, and should not relate to the failures on infrastructure/tests.

File: tools/wptrunner/wptrunner/formatters/chromium.py [hunk 1]
```diff
@@ -258,6 +258,8 @@ def _time_test(self, test_name, data):
     def suite_start(self, data):
         if self.start_timestamp_seconds is None:
             self.start_timestamp_seconds = self._get_time(data)
+        if 'run_info' in data:
+            self.flag_specific = data['run_info'].get('flag_specific') or ''
```
Comment (jcscottiii) [inline tools/wptrunner/wptrunner/formatters/chromium.py:262]:
```suggestion
            self.flag_specific = data['run_info'].get('flag_specific', '')
```

**Comment by jcscottiii** [comment]
I don't think it is related either. But I had one small nit.

File: tools/wptrunner/wptrunner/formatters/chromium.py (hunk 1 above)
Comment (jcscottiii) [reply tools/wptrunner/wptrunner/formatters/chromium.py:262]: Currently if data['run_info'].get('flag_specific') was equal to something falsey like `0`, or `[]`, `data['run_info'].get('flag_specific') or ''` would evaluate to `''`.

**Comment by WeizhongX** [comment]
> I don't think it is related either. But I had one small nit.

Updated, thanks!

**Pull request merged**